Autopilot/Src/IMUThreadInterface.cpp

```diff
@@ -1,12 +1,12 @@
 #include "IMUThreadInterface.h"
 #include "imu.hpp"
 
-void IMUThreadInterfaceInit(void) {
-    static IMU &imuobj = BMX160::getInstance();
-    imuobj.Begin_Measuring();
-}
+static IMU &imuobj = BMX160::getInstance();
```

Comment:
Correct syntax is `static IMU& imuobj = ...;`. Will include this in styleguide when time comes tho